addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1828,17 +1871,10 @@ var SnippetsMenu = Widget.extend({
      * @param {OdooEvent} ev
      */
     _onUpdateCustomizeElements: function (ev) {
-        while (this.customizePanel.firstChild) {
-            this.customizePanel.removeChild(this.customizePanel.firstChild);
-        }
-        ev.data.customize$Elements.forEach($el => {
-            this.customizePanel.appendChild($el[0]);
+        this._updateLeftPanelContent({
+            content: ev.data.customize$Elements,
+            tab: ev.data.customize$Elements.length ? tabs.OPTIONS : tabs.BLOCKS,
```

Comment:
Looks like I forgot to update these instances to `this.tabs` when I refactored the code to be extended from outside rather than have the website functionality inside the web_editor. Sorry!